jsx.d.ts

```diff
@@ -41,7 +41,7 @@ declare namespace JSX {
     tabindex?: undefined | number | string;
     title?: undefined | string;
     translate?: undefined | string | boolean;
-
+    attrs?: undefined | string;
```

Comment:
We should come up with a custom name to clarify to the end user is using a unsafe attribute. I'm not sure if `__rawAttributes` is the better alternative here.

```suggestion
    __rawAttributes?: undefined | string;
```